python/tvm/contrib/graph_runtime.py

```diff
@@ -43,24 +43,34 @@ def create(graph_json_str, libmod, ctx):
         be used as this purpose. All context should be given for heterogeneous
         execution.
 
+    force_local_graph_runtime : bool
+        When ctx contains only rpc contexts, the GraphRuntime is typically instantiated
```

Comment:
This seems to be a quite specialized code path to me. Is it possible to just avoid use graph_runtime.create and use another function(in utvm) to create the Module, then wrap it into GraphModule?

Reply:
We are migrating away from graph_runtime.create, and instead relying on the more general module-based interface  - https://github.com/apache/incubator-tvm/blob/master/tests/python/unittest/test_runtime_module_based_interface.py#L102

The idea is to remove layer of abstractions and only make GraphModule a thin wrapper. Without having to couple the decision of module creation with the GraphModule wrapping.

So it is preferrable to not complicate the graph_runtime.create. It would be useful to have a function in micro tvm that creates the module, then call into GraphModule to wrap the resulting constructed module.

Reply:
reverted changes to graph_runtime.create, moved to `tvm.micro.create_local_graph_runtime`.